### PR TITLE
adding a padding around the close button

### PR DESCRIPTION
**Pull request by pr-Mais**

### CODE REVIEW

File: lib/super_tooltip.dart [hunk 1]
```diff
@@ -999,6 +1015,32 @@ EdgeInsets _getTooltipMargin({
...
+
+    case PreferredDirection.left:
+      return EdgeInsets.only(top: top);
+
+    case PreferredDirection.right:
+      return EdgeInsets.only(top: top);
+
+    default:
+      throw ArgumentError(preferredDirection);
+  }
```
Comment (escamoteur) [inline lib/super_tooltip.dart:1041]: Why do you return the same in  all cases?

File: lib/super_tooltip.dart [hunk 2]
```diff
@@ -87,7 +87,6 @@ class SuperTooltip extends StatefulWidget {
     this.touchThroughAreaShape = ClipAreaShape.oval,
     this.touchThroughAreaCornerRadius = 5.0,
     this.touchThrougArea,
-    // FIXME: Why does a 0.0 borderwidth still show a border?
```
Comment (escamoteur) [inline lib/super_tooltip.dart:90]: Have you tried this? does a borderWith 0 still show a border?

**Review by escamoteur — COMMENTED** [review]
Hi, Mais, see my two comment.
I honestly don't understand some of the 'TD' points in the code. I would need to dig deeper in the code for that.
Cheers & Thanks a lot
Thomas

File: lib/super_tooltip.dart (hunk 1 above)
Comment (pr-Mais) [reply lib/super_tooltip.dart:1041]: I wanted to give a top padding in all 4 directions

File: lib/super_tooltip.dart (hunk 2 above)
Comment (pr-Mais) [reply lib/super_tooltip.dart:90]: Yes I tried, it doesn't

**Comment by pr-Mais** [comment]
I couldn't get them either, thanks for the review!

File: lib/super_tooltip.dart (hunk 1 above)
Comment (escamoteur) [reply lib/super_tooltip.dart:1041]: yes, but if its the same in all direction, what is the `switch` then good for?

Comment (pr-Mais) [reply lib/super_tooltip.dart:1041]: True, will need to update it and get rid of `switch`

**Pull request merged**